features/support/functions.rb

```diff
@@ -134,8 +134,11 @@ def insert_title_with_prices_paid(price_paid_hash)
 end
 
 def insert_multiple_titles(number_of_titles)
+  x = 1
```

Comment:
On second thoughts, it is actually quite useful to see the progress while you're waiting for 51 titles to be created. Maybe we leave it in!